apps/odyssey-stats/src/styles/wp-admin.scss

```diff
@@ -89,4 +89,13 @@
 	ul.wp-submenu-wrap {
 		margin-left: 0;
 	}
+
+	&.wp-core-ui .button.is-secondary {
```

Comment:
Why is this override applied here so broadly?

Reply:
Perhaps it was meant to be nested in a `.stats` selector or something similar? This is quite bad since it will affect all the secondary buttons where the app is loaded.

Reply:
The `Cancel` button is on a popup, so we weren't able to use the `.stats` selector. However you are right, we should limit the scope to the popup rather than everything.